packages/babel-generator/src/generators/jsx.js

```diff
@@ -1,8 +1,9 @@
 export function JSXAttribute(node: Object) {
   this.print(node.name, node);
   if (node.value) {
+    const raw = node.extra && node.extra.raw;
     this.token("=");
-    this.print(node.value, node);
+    this.print(raw || node.value, node);
```

Comment:
I think you can use getPossibleRaw the same way we do for strings: https://github.com/babel/babel/blob/master/packages/babel-generator/src/generators/types.js#L139, if this PR is in fact accepted.